src/lib/y2storage/disk_analyzer.rb

```diff
@@ -355,31 +310,38 @@ def find_windows_partitions
       windows_partitions
     end
 
+    # Checks whether the architecture of the system is supported by
+    # MS Windows
+    #
+    # @return [Boolean]
+    def windows_architecture?
+      # Should we include ARM here?
+      Yast::Arch.x86_64 || Yast::Arch.i386
+    end
+
     # Check if device name 'partition' is a MS Windows partition that could
     # possibly be resized.
     #
-    # @param partition [string| device name of the partition to check
+    # @param partition [::Storage::Partition] partition to check
     #
     # @return [Boolean] 'true' if it is a Windows partition, 'false' if not.
     #
     def windows_partition?(partition)
-      return false unless WINDOWS_PARTITION_IDS.include?(partition.id)
-      return false unless Arch.x86_64 || Arch.i386
```

Comment:
Sure this shortcut is no longer desired for non-PC architectures? How expensive are the subsequent checks? The idea of checking for PC arch here was to avoid them when they clearly don't make any sense (ARM, S/390).